containers/codespaces-linux/.devcontainer/base.Dockerfile

```diff
@@ -56,7 +56,8 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get autoremove -y && apt-get clean -y
 
 # Install Python, PHP, Ruby utilities
-RUN bash /tmp/scripts/python-debian.sh "none" "/opt/python/latest" "${PIPX_HOME}" "${USERNAME}" "true" \
+RUN apt-get install -yq python3-dev \
```

Comment:
Should we add this to the python-debian script instead?

Reply:
Do you mean add it to the `PREREQ_PKGS`? Since we get Python from the Oryx image, we don't build from source meaning we don't install those prereqs. If you mean add it elsewhere, that would work but I assumed it was a Codespaces specific change so I only wanted to make it affect us.

Reply:
Ah, ok. In that case this seems reasonable.

Reply:
This section doesn't otherwise install OS packages and would need a "apt-get clean -y" added at a minimum.  It may be easier to just add it to this line above:

```
 # Verify expected build and debug tools are present
 apt-get -y install build-essential cmake cppcheck valgrind clang lldb llvm gdb 
```